app/helpers/sticky_newsletter_visibility.rb

```diff
@@ -0,0 +1,15 @@
+module StickyNewsletterVisibility
+  def display_sticky_newsletter_on_this_slug?
+    @slug = request.url.gsub request.base_url, ''
```

Comment:
@nmiddleweek this file contains the vital details we need about the Sticky Newsletter. It houses all that we need. 

(https://github.com/moneyadviceservice/frontend/blob/6737_sticky-footer-visibility/features/support/ui/sections/sticky_newsletter.rb

```
require_relative '../section'

module UI::Sections
  class StickyNewsletter < UI::Section
    element :subscription_email, "input[name='subscription[email]']"
    element :send_me_money_advice, ".button--newsletter"
    element :close_button, '.news-signup-sticky__close'
  end
end
```